assets/src/dashboard/app/api/useApiAlerts.js

```diff
@@ -71,6 +77,12 @@ function useApiAlerts() {
     }
   }, [settingsError, showSnackbar]);
 
+  useEffect(() => {
+    if (settingSaved) {
+      debouncedShowSnackbar(SUCCESS.SETTINGS.UPDATED);
+    }
+  }, [debouncedShowSnackbar, settingSaved]);
```

Comment:
This is good! I feel like it would be nice to show something other than `Setting saved` when a poster image is deleted.

Reply:
yeah... i think that's a good thing to separate out purely because copy has some back and forth. i think just showing that the update was successful will be a big win to start with.